packages/universal-router-sdk/src/utils/routerCommands.ts

```diff
@@ -143,16 +143,16 @@ const ABI_PARAMETER: Record<CommandType, any> = {
     'address recipient, uint256 amountOut, uint256 amountInMax, bytes path, bool payerIsUser'
   ),
```

Comment:
yup good catch here the swaps were failing cos v2 takes address array

Reply:
right, the v2/v3 abi not the same, thanks to the strong type viem given, we found it at typecheck stage